mmv1/products/transcoder/JobTemplate.yaml

```diff
@@ -51,7 +51,8 @@ properties:
   - !ruby/object:Api::Type::String
     name: "name"
     description: |
-      The resource name of the job template. Format: projects/{project_number}/locations/{location}/jobTemplates/{jobTemplate}
+      The resource name of the job template.
+    default_from_api: true
```

Comment:
Instead of default from API, I think you could use a custom flattener to suppress any diffs from the name field:

```
    custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
```

This means that if the user supplies a short name but the API returns a longer version of that name then the custom flattener will ignore the additional info